src/pages/App.js

```diff
@@ -65,6 +67,12 @@ export default function App() {
       <p>
         {foods.map((elem) => <li>{elem.description}</li>)}
       </p>
+      <Row>
+        <AnchorButton href="/app" text="Profile" />
```

Comment:
I think the `href` here is meant to be `"/profile"`.

Reply:
Yes thanks for catching this.